VideoWeb/VideoWeb/ClientApp/src/app/security/idp-selection/idp-selection.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { AdalService } from 'adal-angular4';
+import { Logger } from 'src/app/services/logging/logger-base';
+import { ReturnUrlService } from 'src/app/services/return-url.service';
+import { pageUrls } from '../../shared/page-url.constants';
+
+@Component({
+    selector: 'app-idp-selection',
+    templateUrl: './idp-selection.component.html'
+})
+export class IdpSelectionComponent implements OnInit {
+    identityProviders = {
+        ejud: {
+            url: ''
+        },
+        vhaad: {
+            url: '/' + pageUrls.Login
+        }
+    };
+
+    selectedProvider: string;
+    submitted = false;
+
+    constructor(
+        private adalSvc: AdalService,
+        private route: ActivatedRoute,
+        private router: Router,
+        private returnUrlService: ReturnUrlService,
+        private logger: Logger
+    ) {}
+
+    ngOnInit(): void {
+        if (this.isLoggedIn()) {
+            const returnUrl = this.returnUrlService.popUrl() || '/';
+            try {
+                this.logger.debug(`[Login] - User is authenticated. Returning to ${returnUrl}`);
```

Comment:
might be helpful to update start the logger message with the component name

Reply:
done thanks :)